app/javascript/components/Calendar/index.js

```diff
@@ -21,7 +23,7 @@ const styles = {
 }
 
 // Custom components given to BigCalendar
-const calendarComponents = (currentUser, offices, filters, filterActions) => {
+const calendarComponents = (currentUser, offices, filters, filterActions, t) => {
```

Comment:
would be good practice to give `t` a more descriptive name for readability.

Reply:
I think it's standard to have `t` as the translation function ie. the API gives the example: https://www.i18next.com/overview/api

What do you think?

Reply:
yeah , I wasn't aware that this prop was added by a third-party. Ignore me :)